ui-components/src/locales/general.json

```diff
@@ -698,8 +698,8 @@
   "languages": {
     "en": "English",
     "es": "Español",
-    "zh": "中文",
-    "vi": "Tiếng Việt"
```

Comment:
Isn't this just the translation of the languages? I don't think you need to remove zh/vi since that would cause more conflicts in this file compared to upstream.

Reply:
I wonder if it will still cause a merge conflict because we added new strings at the end? if it causes a merge conflict either way, it's probably best to just delete them so we don't have strings that are not applicable (adds to code complexity / maintenance overhead).

Reply:
I'd rather leave it in there. so it doesn't show up in diffs against bloom.

Reply:
Done!